docs/source/changelog.rst

```diff
@@ -44,6 +78,20 @@ Nbconvert 6.0 is a major release of nbconvert which includes many significant ch
 
 The ``--to slides`` exporter is still supported for convenience.
 
+- Inkscape 1.0 is now supported, which had some breaking changes that prevented 5.x versions of nbconvert from converting documents on some systems that updated.
+
+- Support for ``--to webpdf`` for PDF rendering via a browser client was added as an alternative export to pdf from the standard LaTeX -> PDF conversion path.
```

Comment:
We were already mentioning the webpdf exporter above.

Reply:
Whoops -- fixed